packages/FormElement/src/components/Content/Content.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ContentContainerStyled from "./Content.styles";
+
+const propTypes = {
+  children: PropTypes.oneOfType([PropTypes.func, PropTypes.node]).isRequired,
+  idForLabel: PropTypes.string,
+  refLabel: PropTypes.shape({ current: PropTypes.instanceOf(Object) }),
+  ariaDescribedBy: PropTypes.string,
+};
+
+const defaultProps = {
+  idForLabel: "",
+  ariaDescribedBy: "",
+  refLabel: null,
+};
+
+function Content(props) {
+  const { children, idForLabel, refLabel, ariaDescribedBy, ...moreProps } = props;
+
+  if (!children) {
+    return null;
+  }
+
+  return (
+    <ContentContainerStyled data-pka-anchor="formElement.content" {...moreProps}>
+      {typeof children === "function" ? children({ idForLabel, refLabel, ariaDescribedBy }) : children}
```

Comment:
What if in the `else` case here we clone the `children` node (if there's only one) and add `idForLabel`, `refLabel`, `ariaDescribedBy`? Unless we're provided an explicit prop to flip that off?

Reply:
Because how do we know that that single component needs those props? For example only the `listBox` needs the `refLabel`. And perhaps there is only a single checkbox component, which also doesn't need the `idForLabel` etc. I would rather not be checking the type or displayName of the component in this component either as thats the point of this PR. To decouple the component type from what we can provide in the component...

Reply:
Might be a good idea to have a custom propType that will check the `children` prop of `Content` and throw a console warning (in dev env) if the type is not a function – it will still seem like it's working just fine without the `id` and `aria-describedby` so consumers might not realize their implementation is not accessible.  But that can be done in another PR.

cc @nahumzs